public/index.html

```diff
@@ -4,10 +4,11 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" type="text/css" href="https://cloud.typography.com/7761236/7617792/css/fonts.css" />
```

Comment:
This probably needs to be configurable. We can depend on their servers for dev, but we probably want to serve the production version from our own servers. No need to fix in this PR, but we've said that a few times now, so let's start actually filing tickets when we do that.

Reply:
Yes, they very specifically recommend that you serve it yourself in production. They recommend we shift when we're getting ready for people to actually use the app: https://dashboard.typography.com/user-guide/moving-to-production